Sources/WebSocketKit/WebSocketClient.swift

```diff
@@ -34,7 +35,23 @@ public final class WebSocketClient: Sendable {
         /// Maximum frame size after aggregation.
         /// See `NIOWebSocketFrameAggregator` for details.
         public var maxAccumulatedFrameSize: Int
-
+        /// Per Message Compression Extensions Config, if any.
+        /// See `PMCE.PMCEConfig` for details.
+        public var deflateConfig:PMCE.DeflateConfig?
+        
+        // new init to support passing in PMCE.PMCEConfig
+        public init(deflateConfig:PMCE.DeflateConfig?,
+                    tlsConfiguration:TLSConfiguration? = nil,
+                    maxFrameSize:Int = 1 << 14) {
+            
+            self.tlsConfiguration = tlsConfiguration
+            self.maxFrameSize = maxFrameSize
+            self.minNonFinalFragmentSize = 0
+            self.maxAccumulatedFrameCount = Int.max
+            self.maxAccumulatedFrameSize = Int.max
```

Comment:
I don't think the max frame sizes should _ever_ be set to `.max` implicitly.

Reply:
that might be a good catch. Not sure where I saw that from, thought I pulled it from elsewhere in the codebase.
Do you know what default should be?

Reply:
I feel like I copied it from elsewhere, but could have pulled it from somehwere... I just looked. I got it from the pre existing TLS  init I was basing adding pmce arg on ```public init(
            tlsConfiguration: TLSConfiguration? = nil,
            maxFrameSize: Int = 1 << 14
        ) ``` so I was using vapor as the guide as far as I could tell.
I do see ur point of the caller should set that but I see no way for them to pass it on init.
Seeing as it's all sendable now, shouldn't those be loop bound  to be mutable? That would prevent them from chasing it after the fact.
Im surprised it doesnt warn me. as it did on WebSocket having mutable state while being sendable. am confused.